src/script/repositories/calling/CallingRepository.ts

```diff
@@ -1674,6 +1680,11 @@ export class CallingRepository {
       if (call.getSelfParticipant().sharesScreen()) {
         this.stopScreenShare(call.getSelfParticipant(), call.conversation, call);
       }
+
+      // If the user is not part of the conversation, the call must be removed from the state
+      if (reason === LEAVE_CALL_REASON.USER_IS_REMOVED_FROM_CONVERSATION) {
```

Comment:
After the call is terminated, it is ended via AVS (see the last line of code). AVS then triggers an `onClose` event. The call state is then cleaned up there in this event handler. Why is it cleaned up this particular case here now? And what makes it so special?